charts/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -123,20 +122,15 @@ spec:
         {{- end }}
         command:
 {{ toYaml .Values.sysctlImage.command | indent 10 }}
-        securityContext:
-          runAsNonRoot: false
-          privileged: true
-          runAsUser: 0
```

Comment:
Do we not need elevated privileges for initContainers with ```sysctlImage.enabled```?   I'm not very familiar with using this option but I can see certain task needing this.

Reply:
I am going to remove this entirely. sysctl shouldn't really be used anymore this way.